src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -509,40 +509,63 @@ public String getAudioCodec() {
 	@Override
 	public String toString() {
 		StringBuilder result = new StringBuilder();
-		result.append("id: ");
-		result.append(getId());
-		result.append(", lang: ");
-		result.append(getLang());
+		if (!getLang().equals("und")) {
+			result.append("Id: ").append(getId());
+			result.append(", Language Code: ").append(getLang());
+		}
 
 		if (isNotBlank(getAudioTrackTitleFromMetadata())) {
-			result.append(", audio track title from metadata: ");
-			result.append(getAudioTrackTitleFromMetadata());
+			if (result.length() > 0) {
+				result.append(", ");
+			}
+			result.append("Audio Track Title From Metadata: ").append(getAudioTrackTitleFromMetadata());
 		}
 
-		result.append(", audio codec: ");
-		result.append(getAudioCodec());
-		result.append(", sample frequency:");
-		result.append(getSampleFrequency());
-
-		if (getAudioProperties() != null && getAudioProperties().getNumberOfChannels() != 0) {
-			result.append(", number of channels: ");
-			result.append(getAudioProperties().getNumberOfChannels());
+		if (result.length() > 0) {
+			result.append(", ");
+		}
+		result.append("Audio Codec: ").append(getAudioCodec());
+		if (getSampleFrequency() != null) {
+			result.append(", Sample Frequency: ").append(getSampleFrequency());
+		}
+		if (getBitRate() != 0) {
+			result.append(", Bitrate: ").append(getBitRate());
+		}
+		if (getBitsperSample() != 0 && getBitsperSample() != 16 ) {
```

Comment:
@Nadahar I guess that we don't share debugging view.

As they are MediaInfo issues, i don't really see the use of loosing space, or overcharging the log with wrong informations, when the only fact that they are not displayed in the log just mean exactly the same thing without overcharging the log.

That's said it's only my point of view